assets/css/edd-admin.css

```diff
@@ -2431,6 +2414,56 @@ body.download_page_edd-reports {
 	margin-bottom: 20px;
 }
 
+div.edd-reports-export {
+	display: grid;
+	display: -ms-grid;
+	grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+	grid-gap: 20px;
+}
+
+.edd-reports-wrapper .postbox {
+	margin-bottom: 0;
+}
+
+.edd-reports-export .edd-from-to-wrapper {
+	display: flex;
+	margin-right: 0;
+	margin-bottom: 16px;
+	width: 100%;
+}
+
+.edd-reports-export .edd-from-to-wrapper input {
+	width: 100%;
+}
+
+.edd-reports-export .edd-from-to-wrapper span {
+    flex-grow: 1;
+}
+
+.edd-import-export-form {
+	display: flex;
+	flex-direction: column;
+	flex-wrap: wrap;
+}
+
+div.edd-to-and-from-container {
+	display: grid;
+	display: -ms-grid;
+	grid-template-columns: 1fr 1fr;
+	grid-gap: 8px;
+}
+
+span.edd-to-and-from--seperator {
```

Comment:
I like having the different class, but there is `a rat` in separate--spelling should be updated here and in the PHP.

Reply:
Good catch.  Thank you.